src/ds5/ds5-auto-calibration.h

```diff
@@ -58,8 +54,40 @@ namespace librealsense
             RS2_DSC_STATUS_NOT_CONVERGE = 4, /**< For tare calibration only*/
             RS2_DSC_STATUS_BURN_SUCCESS = 5,
             RS2_DSC_STATUS_BURN_ERROR = 6,
-            RS2_DSC_STATUS_NO_DEPTH_AVERAGE = 7,
+            RS2_DSC_STATUS_NO_DEPTH_AVERAGE = 7
         };
+
+        enum speed
```

Comment:
Even though it is namespace-isolated, using `auto_calib_speed` can be better to avoid potential collisions